src/main/kotlin/dev/stalla/dom/DomParsingExtensions.kt

```diff
@@ -133,14 +127,15 @@ internal fun Node.toHrefOnlyImageBuilder(imageBuilder: HrefOnlyImageBuilder): Hr
  * @return The [personBuilder] populated with the DOM node contents.
  */
 @InternalApi
+@Suppress("LoopWithTooManyJumpStatements")
 internal fun Node.toPersonBuilder(personBuilder: PersonBuilder, namespace: FeedNamespace? = null): PersonBuilder {
     for (child in childNodes.asListOfNodes()) {
         if (child !is Element) continue
         if (!namespace.matches(child.namespaceURI)) continue
         val value: String? = child.textOrNull()
```

Comment:
Had no good idea how to simplify this, so `@Suppress`ed

Reply:
I don't really see a way either, a suppress is good enough.